modules/components/print-style.js

```diff
@@ -0,0 +1,46 @@
+var React = require('react');
+
+var Style = require('./style.js');
+var printStyles = require('../print-styles.js');
+
+var PrintStyle = React.createClass({
```

Comment:
I was keeping in line the with `Style` component but I agree that `PrintStyleSheet` actually encapsulates the purpose of this component better than `PrintStyle`.